modules/cloudfront-to-s3-to-cloudwatch/bucket.tf

```diff
@@ -9,6 +9,33 @@ resource "aws_s3_bucket" "bucket" {
   count = var.create_bucket ? 1 : 0
 
   bucket = local.bucket_name
+  versioning {
+    enabled = var.enabled
```

Comment:
same issue here related to non descriptive values,
lets have group of configs, something like
s3_bucket_options = {
 versioning_enabled =optional(string, false)
 logging= optional(list(any), [])
 ...
}